lib/mirage.ml

```diff
@@ -1680,7 +1715,14 @@ let configure i =
   let ctx = Info.context i in
   let target = Key.(get ctx target) in
   Log.info (fun m -> m "Configuring for target: %a" Key.pp_target target);
+  let gdb = Key.(get ctx gdb) in
   let opam_name = unikernel_name target name in
+  let libs = Info.libraries i in
+  let _ = match target with
+    | `Ukvm -> R.ok ()
+    | _ -> (if gdb then Log.warn (fun m -> m "--gdb not supported on this target."));
+      R.ok ()
+  in
```

Comment:
this is slightly overcomplicated, we can use the following instead:
```OCaml
if gdb && target <> `Ukvm then
  Log.warn ... ;
```
(i.e. no need for the `match`, also not to `R.ok ()` in both branches).